app/styles/layouts/auth.css

```diff
@@ -17,6 +17,10 @@
     margin-bottom: 1.5rem;
 }
 
+.gh-signin .form-group.error {
```

Comment:
OH wait nevermind I see what you mean 😄  However, on master there are no specific errors displayed, just "Please fill out the form to sign in." Any specific field validations don't seem to be displayed.